src/main/resources/templates/orderLookup.html

```diff
@@ -19,7 +19,7 @@ <h1 class="display-4">Order Lookup</h1>
     </div>
 </main>
 
-<div class="container">
+<div class="container outerContentWrapper">
```

Comment:
Added the outerContentWrapper CSS class to constrain the content and keep it from overflowing into other elements, such as the footer.